app/build.gradle

```diff
@@ -25,7 +25,7 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation "com.android.support:support-annotations:$rootProject.supportLibraryVersion"
     implementation "com.android.support:appcompat-v7:$rootProject.supportLibraryVersion"
-    implementation "com.android.support:preference-v7:$rootProject.supportLibraryVersion"
+    implementation "com.takisoft.fix:preference-v7:$rootProject.supportLibraryVersion.0"
```

Comment:
Why do we need a third-party library?

Reply:
@CloudyPadmal because modifying the input type in EditTextPrefs is not properly supported across all API levels in the default preference library.

Reply:
@CloudyPadmal refer to this - https://stackoverflow.com/questions/42804842/edittextpreference-only-numeric-value-inputtype-isnt-working

Reply:
Have a different variable name. Here zero is hard coded.